test/wpt/test-url.js

```diff
@@ -11,6 +11,12 @@ runner.setScriptModifier((obj) => {
     // created via `document.createElement`. So we need to ignore them and just
     // test `URL`.
     obj.code = obj.code.replace(/\["url", "a", "area"\]/, '[ "url" ]');
+  } else if (obj.filename.includes('urlsearchparams-constructor.any.js')) {
+    // Ignore test named `URLSearchParams constructor, FormData.` because we do
+    // not have `FormData`.
```

Comment:
If we decided to add FormData later someone might miss removing this. It would be worthwhile making this replacement dependent on whether FormData is undefined.

Reply:
Done. I've add `typeof FormData === 'undefined'`.

Reply:
/ping @jasnell

Reply:
/ping @jasnell